packages/nimble-components/src/table/testing/table.pageobject.ts

```diff
@@ -604,6 +618,19 @@ export class TablePageObject<T extends TableRecord> {
         return rows.item(rowIndex);
     }
 
+    private getRowById(recordId: string): TableRow {
+        const row: TableRow | null = this.tableElement.shadowRoot!.querySelector(
+            `nimble-table-row[record-id="${recordId}"]`
```

Comment:
Since we don't have many restrictions on record ID / column ID formats, they can contain double-quotation marks which would result in an invalid selector here (and in `getCellById`).
We probably need to add escaping - it looks like [CSS.escape()](https://developer.mozilla.org/en-US/docs/Web/API/CSS/escape_static) should work.